packages/cice5/package.py

```diff
@@ -18,8 +13,27 @@ class Cice5(MakefilePackage):
     maintainers("harshula", "anton-seaice")
     license("BSD-3-Clause", checked_by="anton-seaice")
 
-    version("access-om2", branch="master", preferred=True)
-    version("access-esm1.6", branch="access-esm1.6")
+    version("stable", branch="master", preferred=True)
+    version("2025.03.001", tag="2025.03.001", commit="4cc7abd9875153a8114647fd3317a31993fdf80d")
+    version("access-esm1.6-2025.09.000", tag="access-esm1.6-2025.09.000", commit="16e26c8585eca4cb2ba97eb6462ce3b803a8b46c")
+
+    variant(
+        "model",
+        default="access-om2",
+        values=("access-om2", "access-esm1.6"),
+        description="Which model this build is coupled with"
+    )
+
+    conflicts(
+        "model=access-esm1.6",
+        when="@2025:",
+        msg="Version is not compatible with model=access-esm1.6"
```

Comment:
This is enforcing a restriction that the `acess-esm1.6` model can only use versions `2024.*` and earlier?